doc/guides/writing-and-running-benchmarks.md

```diff
@@ -298,6 +299,11 @@ chunk     encoding       mean confidence.interval
 
 ![compare tool boxplot](doc_img/scatter-plot.png)
 
+### Running Benchmarks on the CI
+
+To see the performance impact of a Pull Request by running benchmarks on
+the CI, check out [How to: Running core benchmarks on Node.js CI](benchmark-ci).
```

Comment:
This link is 404. `[How to: Running core benchmarks on Node.js CI][benchmark-ci]`?